runtime/compiler/codegen/J9CodeGenerator.hpp

```diff
@@ -377,6 +377,15 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
     */
    void setSupportsInlineStringCaseConversion() { return _j9Flags.set(SupportsInlineStringCaseConversion);}
 
+   /** \brief
+    *    Determines whether the code generator supports inlining of java/lang/String.indexOf()
+    */
+   bool getSupportsInlineStringIndexOf() { return _j9Flags.testAny(SupportsInlineStringIndexOf);}
+   /** \brief
```

Comment:
Can we have a blank line between the definitions?

Reply:
8ffd466